### DISCUSSION TITLE
Welcome to twidge Discussions!

### DISCUSSION THREAD

**Discussion by VarunPotti** [VarunPotti/twidge#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by heshangambhir** [comment]
Hey, everyone, this is Heshan I'm a 15-year-old software developer and entrepreneur currently working on my startup  Disquzz A Cross Platform CBT App Based On AI. I create content on Twitter around web development, web3, entrepreneurship, freelancing, podcasting, and much more let's connect!
I am excited to work on twidge! 

https://twitter.com/heshangambhir
